src/AsyncGenerator.yml

```diff
@@ -4,8 +4,6 @@
   applyChanges: true
   analyzation:
     methodConversion:
-    - conversion: Ignore
-      hasAttributeName: ObsoleteAttribute
```

Comment:
As for now we want to generate async counterparts for obsoleted methods (as they were here before the methods were marked obsolete)